docs/source/content/api.md

```diff
@@ -73,6 +71,50 @@ the format of valid observations is specified by `env.observation_space`.
 In the example above we sampled random actions via `env.action_space.sample()`. Note that we need to seed the action space separately from the 
 environment to ensure reproducible samples.
 
+### Change in env.step API
+
+Previously, the step method returned only one boolean - `done`. This is being deprecated in favour of returning two booleans `terminated` and `truncated`.
+
+Environment termination can happen due to any number of reasons inherent to the environment eg. task completition, failure etc. This is distinctly different from an episode of the environment ending due to a user-set time-limit, referred to as 'truncation'. 
+
+`terminated` signal is set to `True` when the core environment terminates inherently because of task completition, failure etc. 
+`truncated` signal is set to `True` when the episode ends specifically because of a time-limit not inherent to the environment. 
+
+It is possible for `terminated=True` and `truncated=True` to occur at the same time when termination and truncation occur at the same step. 
+
+#### Motivation 
+This is done to remove the ambiguity in the `done` signal. `done=True` does not distinguish between the environment terminating and the episode truncating. This problem was avoided previously by setting `info['TimeLimit.truncated']` in case of a timelimit through the TimeLimit wrapper. However, this forces the truncation to happen only through a wrapper, and does not accomodate environments which truncate in the core environment. 
```

Comment:
"However, this forces the truncation to happen only through a wrapper"

This is not true, any env can set `info['TimeLimit.truncated'] = True`...

Reply:
Oh of course, my bad!